Makefile

```diff
@@ -1062,15 +1070,12 @@ endif
 # Packaging components
 #
 ##############################
-
-.PHONY: package
-package:
-	$(V1) cd $@ && $(MAKE) --no-print-directory $@
-
-.PHONY: standalone
-standalone:
+PACKAGE_TARGETS = package_ground package_flight package_installer package_matlab package_all standalone package_info
+PACKAGE_TARGETS += package_ground_compress package_flight_compress package_matlab_compress package_all_compress
```

Comment:
What do you mean? use it only as the .PHONY and target instead of the complete list?